packages/xfsprogs/xfsprogs.spec

```
@@ -14,6 +16,9 @@ BuildRequires: %{_cross_os}libblkid-devel
 
 Requires: %{_cross_os}liburcu
 Requires: %{_cross_os}libinih
+Requires: (%{_cross_os}kernel-5.10-mkfs-confs if %{_cross_os}kernel-5.10)
+Requires: (%{_cross_os}kernel-5.15-mkfs-confs if %{_cross_os}kernel-5.15)
+Requires: (%{_cross_os}kernel-6.1-mkfs-confs if %{_cross_os}kernel-6.1)
```

Please just require a generic capability that any kernel package can implement. Otherwise it will be challenging to bring a custom kernel of a different version.

```suggestion
Requires: (%{_cross_os}kernel(mkfs-confs)
```